src/geodata2.erl

```diff
@@ -91,5 +93,13 @@ get_env(App, Key) ->
 id(X) ->
     X.
 
+%% @TODO [RTI-8302] This one could be removed after the IPv4+IPv6 MMDB is definitely used
+is_ipv6_mmdb() ->
+    application:get_env(geodata2, ipv6, false).
+
+%% @TODO [RTI-8302] This one could be removed after the IPv4+IPv6 MMDB is definitely used
```

Comment:
This library is public (it's a fork from somebody else) and those jira tickets are private. I'm not sure if we should leave them here.
As a matter of fact, we might eventually send a PR upstream… and we'll definitely not want them there then.